src/surface_conditions/surface_conditions.jl

```diff
@@ -332,28 +340,33 @@ end
 function surface_temperature(
     ::RCEMIPIISST,
     coordinates::Union{Geometry.LatLongZPoint, Geometry.LatLongPoint},
+    surface_temp_params,
 )
     (; lat) = coordinates
+    (; SST_mean, SST_delta, SST_wavelength_latitude) = surface_temp_params
     FT = eltype(lat)
-    T = FT(300) + FT(1.25) / 2 * cosd(360 * lat / 54)
+    T = SST_mean + SST_delta / 2 * cosd(360 * lat / SST_wavelength_latitude)
     return T
 end
 
-#Plane SST distribution from Wing et al. (2023) https://gmd.copernicus.org/preprints/gmd-2023-235/
+#Box SST distribution from Wing et al. (2023) https://gmd.copernicus.org/preprints/gmd-2023-235/
 function surface_temperature(
     ::RCEMIPIISST,
     coordinates::Union{Geometry.XZPoint, Geometry.XYZPoint},
+    surface_temp_params,
 )
     (; x) = coordinates
+    (; SST_mean, SST_delta, SST_wavelength) = surface_temp_params
     FT = eltype(x)
-    T = FT(300) + FT(1.25) / 2 * cos(2 * FT(pi) * x / 6000)
+    T = SST_mean + SST_delta / 2 * cos(2 * FT(pi) * x / SST_wavelength)
```

Comment:
```suggestion
    T = SST_mean - SST_delta / 2 * cos(2 * FT(pi) * x / SST_wavelength)
```
(It's a minus sign in Eq. 1 in Wing et al. 2023?)